### PR TITLE
Use `go install` to install binary

### PR DESCRIPTION
**Pull request by salaboy**
"Fixes #issue-number" or "Add description of the problem this PR solves"

## Proposed Changes 

- use `go install` instead of `go get`, as this seems to be the new way of doing things.

### CODE REVIEW

File: docs/serving/autoscaling/autoscale-go/README.md [hunk 1]
```diff
@@ -6,7 +6,7 @@ A demonstration of the autoscaling capabilities of a Knative Serving Revision.
 
 1. A Kubernetes cluster with [Knative Serving](../../../install/yaml-install/serving/install-serving-with-yaml.md))
    installed.
-1. The `hey` load generator installed (`go get -u github.com/rakyll/hey`).
+1. The `hey` load generator installed (`go install -u github.com/rakyll/hey`).
```
Comment (psschwei) [inline docs/serving/autoscaling/autoscale-go/README.md:9]:
```suggestion
1. The `hey` load generator installed (`go install github.com/rakyll/hey@latest`).
```

**Review by psschwei — COMMENTED** [review]
Thanks for the fix!

**Review by evankanderson — COMMENTED** [review]
/lgtm
/approve

**Comment by evankanderson** [comment]
I'm fascinated how the git history for this will look, but it looks @salaboy checked the box to allow maintainers to apply patches to his branch, so I merged Paul's suggestion and then approved the mix.

**Comment by psschwei** [comment]
image

image

**Pull request merged**